pages/_document/index.js

```diff
@@ -37,7 +38,6 @@ export default class MyDocument extends Document {
         </Head>
         <body>
           <Main />
-          <NextScript />
```

Comment:
Haha, yeah, I misunderstood you.

And when I said "nothing seems to break" I mean: "I was a bad developer and everything looked fine but nothing actually functioned." lol